docs/src/content/components/datepicker.mdx

```diff
@@ -748,7 +748,21 @@ export default Example
 
 ## Migration guide from v11.x and before to v12.x
 
-This guide is to migrate the old DatePicker to a single datepicker using the new components and functions.
+### Simple
```

Comment:
I would move this to the top of the docs. It seems to be more of the simple common case and less of a migration issue.  Migration hand-holding definitely more useful in the custom date picker case.